packages/server/src/db/seeders/20230607183046-seed-badge.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const animals = [
+  'aap',
+  'beer',
+  'buffel',
+  'eend',
+  'eland',
+  'geit',
+  'giraf',
+  'gorilla',
+  'hond',
+  'kikker',
+  'kip',
+  'koe',
+  'konijn',
+  'krokodil',
+  'kuiken',
+  'luiaard',
+  'narwal',
+  'neushoorn',
+  'nijlpaard',
+  'olifant',
+  'paard',
+  'panda',
+  'papegaai',
+  'pinguïn',
+  'slang',
+  'uil',
+  'varken',
+  'walrus',
+  'walvis',
+  'zebra'
+];
+let badges = [];
+
+for (const animal of animals) {
```

Comment:
Could you move this for loop inside of the `up` function. Now, every time we run seeders this will add unnecessary load.

Reply:
Done :-)